package.json

```diff
@@ -95,8 +95,7 @@
   "dependencies": {
     "@backstage/errors": "workspace:^",
     "@manypkg/get-packages": "^1.1.3",
-    "@types/global-agent": "^2.1.3",
-    "@useoptic/optic": "^1.0.0"
```

Comment:
I don't believe this removal should be part of the PR, right?

Reply:
nice catch